app/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "__MSG_appName__",
   "short_name": "__MSG_appName__",
-  "version": "7.7.0",
+  "version": "7.8.0",
```

Comment:
Could you undo this change? We don't update the version in feature branches as a general rule. We prepare releases in a separate branch.

Reply:
yeh sure